src/Webapi/Webapi__Fetch.res

```diff
@@ -412,13 +413,35 @@ module Request = {
   @get external signal: t => signal = "signal"
 }
 
+module ResponseInit = {
+  type t = responseInit
+
+  @obj
+  external make: (
+    ~status: int=?,
+    ~statusText: string=?,
+    ~headers: headersInit=?,
+    unit,
+  ) => responseInit = ""
+  let make = (
+    ~status: option<int>=?,
+    ~statusText: option<string>=?,
+    ~headers: option<headersInit>=?,
```

Comment:
This can also be a `Js.Dict.t<string>` or `array<(string, string)>`, do you think we should have corresponding functions to accept those?

Reply:
I suppose it needs to accept a `Headers.t` too, hmm.

Reply:
Taking another look, this is how RequestInit currently works and we can discuss anything extra in a followup issue.